### PR TITLE
Feature/Preprints Content Page - Mobile View [PREP-201]

### PR DESCRIPTION
**Pull request by pattisdr**
# Ticket 
https://openscience.atlassian.net/browse/PREP-201

# Purpose
Fixes several mobile-view issues on the preprints content page.

# Changes
- Edit button/logo were cut off on mobile view - removes logo entirely and moves edit button down to next row. Centers all text in header (title, contributors, dates, edit button).
screen shot 2016-12-14 at 12 04 55 pm

- "Download preprint" and version display were overtop of other data - moves to new line.
screen shot 2016-12-14 at 12 05 44 pm

- Distinguish "Primary" file differently in mobile view - different layout for supplementary file browser - mobile view 

screen shot 2016-12-14 at 12 24 08 pm
screen shot 2016-12-14 at 12 25 18 pm

- Change formatting of share icons - move to next line.
screen shot 2016-12-14 at 12 05 49 pm

### CODE REVIEW

**Comment by pattisdr** [comment]
@icereval, so this is the PR you want me to upgrade ember-resolver dependency?

**Comment by brianjgeiger** [comment]
I think it's just "whichever PR fixes the last places where it's wrong".

**Comment by icereval** [comment]
@brianjgeiger is correct, if we can just wrap in @binoculars couple changes I think we'll be g2g and avoid the depreciation.

File: package.json [hunk 1]
```diff
@@ -58,11 +58,11 @@
     "ember-link-action": "0.0.35",
     "ember-load-initializers": "^0.5.1",
     "ember-metrics": "0.6.3",
-    "ember-osf": "git+https://github.com/CenterForOpenScience/ember-osf.git#db96ee083e439ba26b4f62320a4482967b11d1aa",
+    "ember-osf": "git+https://github.com/CenterForOpenScience/ember-osf.git#873b1e360d2762fb8d7c1cb3fada89c4b60b658f",
     "ember-page-title": "3.0.6",
     "ember-power-select": "1.0.0-beta.7",
     "ember-read-more": "0.0.13",
-    "ember-resolver": "2.1.0",
+    "ember-resolver": "^2.0.3",
```
Comment (binoculars) [inline package.json]: Let's leave this pinned at 2.1.0 for now, at least until yarn goes through, to avoid merge conflicts.

**Pull request merged**